docs/content/usage/reports/mitre.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "MITRE ATT&CK report"
+date: 2020-06-24T21:17:18+03:00
+draft: false
+---
+
+The Monkey maps its actions to the [MITRE ATT&CK](https://attack.mitre.org/) knowledge base: It provides a new report with the utilized techniques and recommended mitigations, to help you simulate an APT attack on your network and mitigate real attack paths intelligently.
+
+Watch an overview video:
+
+{{% youtube 3tNrlutqazQ %}}
+
+## How to use the report
+
+The MITRE ATT&CK report is centred around the ATT&CK matrix:
+
+![MITRE Report](/images/usage/reports/mitre-report-0.jpg "MITRE Report")
+
+The Monkey rates your network on the attack techniques it attempted. For each technique, you can get
+
+- **Red**: The Monkey **successfully used** the technique in the simulation. That means your network is vulnerable to this technique being employed.
+- **Yellow**: The Monkey **tried to use** the technique, but didn’t manage to. That means your network isn’t vulnerable to the way Monkey employs this technique.
```

Comment:
**Grey** not explained

Reply:
Also, maybe add colour coat as in the ZT report docs?

Reply:
Done and done